src/components/shell-center-row/shell-center-row.scss

```diff
@@ -27,6 +27,10 @@
   border-0;
 }
 
+:host([collapsed]) {
+  @apply hidden;
```

Comment:
Should we just hide the internals of the component instead of the component itself? Because essentially this is just emulating the `hidden` attribute and they may not play nice together.

Reply:
edit: Saw the edit that this is temporary. If its temporary then I'm ok with it but ideally we don't want to mess with the host in these cases

Reply:
It is temporary, but I don't know how long the rework will take so I might as well do it right so nothing breaks until then.

Reply:
Sorry by the rework I meant @macandcheese thinks we should completely deprecate the `calcite-shell-center-row` component in favor of new top/bottom slots for `calcite-shell`. But idk how long that will take, I assume way past v1. I'm just going to close this PR, it only took a sec to put together and the users should just use `hidden`